packages/@react-spectrum/accordion/chromatic/Accordion.stories.tsx

```diff
@@ -89,5 +89,3 @@ export const WithDisabledDisclosure = {
       </Disclosure>
     </Accordion>)
 };
-
-// TODO: more stories
```

Comment:
Don't we still need more chromatic stories, like a button within a disclosure header and props liek isQuiet?

Reply:
not sure if we're supporting other buttons in the disclosure header for v3. i think that was just a change for s2

Reply:
We don't support `isQuiet` now in v3. Bringing that up with the team to see if we want that added.